### DISCUSSION TITLE
Need guidence on Prepayment flow

### DISCUSSION THREAD

**Discussion by nemishzalavadiya-pirimid** [ONDC-Official/ONDC-FIS-Specifications#22] [Q&A]
From this,
Draft [link](https://github.com/ONDC-Official/ONDC-FIS-Specifications/tree/draft-2.x)
Swagger [link](https://ondc-official.github.io/ONDC-FIS-Specifications/)

Flow UI tab do not have mention of how pre-payment will work? if so request to add example

Note: I have checked discussion in this board & response to use update and on_update protocol.

**Comment by Antriksh-Parmar** [comment]
@92shreyansh

**Comment by 92shreyansh** [comment]
Will update the flow UI for the same soon
